2200088/파리퇴치.py

```diff
@@ -0,0 +1,17 @@
+T = int(input())
+
+for t in range(1,1+T):
+    N, M = map(int, input().split())
+    lst = [list(map(int, input().split())) for _ in range(N)]
+
+    a = 0
+    lst2 = []
+    for i in range(N-M+1): # 1
+        for j in range(N-M+1): #  0 
+            for k in range(i,M+i): # 0 1
+                a += sum(lst[k][j:M+j])
+            lst2.append(a)
+            a = 0
+    lst2.sort()
+    print(f'#{t} {lst2[-1]}')
```

Comment:
깔끔한 풀이네요! 👍
```suggestion
    print(f'#{t} {max(lst2)}')
```
저희는 최댓값만 필요하므로, `max` 함수를 활용해주셔도 될 것 같아요!